SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -10,15 +10,41 @@
 #include "FileHelpers.h"
 
 #include "AssetRegistryModule.h"
+#include "GameFramework/GameModeBase.h"
+#include "GameFramework/WorldSettings.h"
 #include "GeneralProjectSettings.h"
+#include "Misc/ConfigCacheIni.h"
 #include "Misc/ScopedSlowTask.h"
 #include "UObject/StrongObjectPtr.h"
+#include "Engine/WorldComposition.h"
+
+
 
 DEFINE_LOG_CATEGORY(LogSpatialGDKEditor);
 
 #define LOCTEXT_NAMESPACE "FSpatialGDKEditor"
 
 bool FSpatialGDKEditor::GenerateSchema(bool bFullScan)
+{
+	FAssetRegistryModule& AssetRegistryModule = FModuleManager::LoadModuleChecked<FAssetRegistryModule>("AssetRegistry");
```

Comment:
Is LoadModuleChecked safe to run, if it fails does this crash the editor?

Reply:
Also second place we've called this, perhaps make a function or part of an init?